browser-sign-in/ios/Podfile

```diff
@@ -1,13 +1,66 @@
-platform :ios, '13.0'
-inhibit_all_warnings!
+require File.join(File.dirname(`node --print "require.resolve('expo/package.json')"`), "scripts/autolinking")
+require File.join(File.dirname(`node --print "require.resolve('react-native/package.json')"`), "scripts/react_native_pods")
 
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
-require_relative '../node_modules/react-native/scripts/react_native_pods'
+require 'json'
+podfile_properties = JSON.parse(File.read(File.join(__dir__, 'Podfile.properties.json'))) rescue {}
+
+ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['newArchEnabled'] == 'true' ? '1' : '0'
+ENV['EX_DEV_CLIENT_NETWORK_INSPECTOR'] = podfile_properties['EX_DEV_CLIENT_NETWORK_INSPECTOR']
+
+platform :ios, podfile_properties['ios.deploymentTarget'] || '15.1'
+install! 'cocoapods',
+  :deterministic_uuids => false
+
+prepare_react_native_project!
 
 target 'browserSignIn' do
-  config = use_native_modules!
-  use_react_native!(:path => config[:reactNativePath])
+  use_expo_modules!
+
+  if ENV['EXPO_USE_COMMUNITY_AUTOLINKING'] == '1'
+    config_command = ['node', '-e', "process.argv=['', '', 'config'];require('@react-native-community/cli').run()"];
```

Comment:
Is this generated? Because having the `require` on the same line feels weird, but if it is auto-generated, then 🤷🏼‍♀️

Reply:
Yep. Everything in android and ios folders is autogenerated. I made an additional change in android/app/build.gradle to include manifestPlaceHolders but that's all there is to it. ios and android folders aren't worth reviewing